[BUG] Game won;t load after updating mod with mod manager but will work if same mod is downloaded from site, button to toggle seeing updates for mods displays incrorectly.

**Describe the bug**
After updating continuity to 1.02 with mod manager it told me to relaunch and after relaunching the game gave me an error

Also the button that toggles showing mods updates isn;t displaying properly

![image](https://user-images.githubusercontent.com/42551576/140655258-818ae0ee-1d7e-4440-b7df-5ab219920ba4.png)

**To Reproduce**
Steps to reproduce the behavior:
1. Open game and go to mod settings
2. Click on download button
3. Update continuity and restart
4. See error

**Expected behavior**
Load game 

**Please complete the following information:**
 - OS: Windows 10
 - Minecraft version 1.17.1
 - Version modmanager-1.2.0-alpha+1.17

**Logs**
Latest log:
[2021-11-07-6.log](https://github.com/DeathsGun/ModManager/files/7492767/2021-11-07-6.log)


Crash-report (if any):
java.lang.RuntimeException: Could not execute entrypoint stage 'preLaunch' due to errors, provided by 'modmanager'!
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$0(EntrypointUtils.java:51)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:49)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35)
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:134)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:68)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)
Caused by: java.nio.file.NoSuchFileException: D:\Games\MultiMC\instances\1.17.1\.minecraft\mods\continuity-1.0.0+1.17.jar
	at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:85)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
	at java.base/sun.nio.fs.WindowsFileSystemProvider.implDelete(WindowsFileSystemProvider.java:275)
	at java.base/sun.nio.fs.AbstractFileSystemProvider.delete(AbstractFileSystemProvider.java:105)
	at java.base/java.nio.file.Files.delete(Files.java:1146)
	at xyz.deathsgun.modmanager.PreLaunchHook.onPreLaunch(PreLaunchHook.kt:42)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47)
	... 12 more

--- a/src/main/kotlin/xyz/deathsgun/modmanager/PreLaunchHook.kt
+++ b/src/main/kotlin/xyz/deathsgun/modmanager/PreLaunchHook.kt
@@ -39,7 +39,10 @@ class PreLaunchHook : PreLaunchEntrypoint {
         for (file in filesToDelete) {
             logger.info("Deleting {}", file)
             val path = Path(file)
-            Files.delete(path)
+            try {
+                Files.delete(path)
+            } catch (e: Exception) { // Ignore it
+            }
         }
     }
 

